terragrunt/aws/iam/iam.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  readonly_role_name = "GHOIDCCodespacesReadOnlyRole"
+}
+
+module "github_oidc_role" {
+  source = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v4.0.0"
```

Comment:
See comment for S3 bucket. You don't want this to be created in every account, so probabably want to limit this with a count variable. Not as vital now but will be as a production account exists.